aws/elbv2_test.go

```diff
@@ -25,20 +25,36 @@ func createTestELBv2(t *testing.T, session *session.Session, name string) elbv2.
 		assert.Fail(t, "Needs at least 2 subnets to create ELBv2")
 	}
 
-	subnet1 := *subnetOutput.Subnets[0]
-	subnet2 := *subnetOutput.Subnets[1]
+	var firstSubnet ec2.Subnet
+	var secondSubnet ec2.Subnet
+
+	// ensure we select subnets in different availability zones
```

Comment:
Any time you find yourself marking of blocks of code with a comment, you're typically better off extracting that code into a method with a descriptive name, such as `getSubnetsInDifferentAZs(...)`.